debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/OracleStreamingChangeEventSourceMetrics.java

```diff
@@ -694,6 +694,10 @@ public void calculateTimeDifference(OffsetDateTime databaseSystemTime) {
         LOGGER.trace("Current time {} ms, database difference {} ms", now.toEpochMilli(), timeDiffMillis);
     }
 
+    public long getDatabaseOffsetSeconds() {
```

Comment:
Can we use a more semantical type for representing `offsetSeconds`, e.g. `java.time.ZoneOffset`?